src/renderer/components/ViewContent/ViewContent.tsx

```diff
@@ -0,0 +1,11 @@
+import { BoxProps } from '@mui/material';
+import React from 'react';
+import { BackgroundWrapper, ContentBox } from './style';
+
+const ViewContent: React.FC<BoxProps> = (props) => (
+  <BackgroundWrapper {...props}>
+    <ContentBox>{props.children}</ContentBox>
+  </BackgroundWrapper>
```

Comment:
@LostInStatic I have never considered this before but if the `props` are spread like that will it also spread the `children` to the `BackgroundWrapper` initially and then override it with the actual `children` passed in jsx? 🤔 

If we remove the `ContentBox` the `children` will still render to the `BackgroundWrapper`, albeit in a chaotic non styled way of course,  since `children` is just a prop in React. This just makes me wonder how React handles this internally and if there are any unnecessary assignments done under the hood because of the double passing of children. 

Can't find any immediate answers to this with a super quick search so perhaps it is just best to mitigate this by separating the `children` from other props:

```tsx
const ViewContent: React.FC<BoxProps> = ({ children, ...props }) => (
  <BackgroundWrapper {...props}>
    <ContentBox>{children}</ContentBox>
  </BackgroundWrapper>
);
```

Reply:
Yup, good point - my guess is that elements between the tags override the children passed as prop. If I try to set them explicitly ( `<BackgroundWrapper children={props.children} {...props}>` ), React protests about setting children twice. 

Anyway, I've applied your suggestion.